routes/v1.js

```diff
@@ -194,5 +195,12 @@ apiRoutes.route('/articles/:slug/comments/:id')
     CommentController.deleteComment
   );
 
+apiRoutes.route('/comments/:id/like')
+  .put(
```

Comment:
Can you please explain why you're using a put method here instead of post?

Reply:
Updated to Post method as discusses.